scripts/react_native_pods.rb

```diff
@@ -209,7 +209,8 @@ def react_native_post_install(installer, react_native_path = "../node_modules/re
     flipper_post_install(installer)
   end
 
-  package = JSON.parse(File.read(File.join(react_native_path, "package.json")))
+  package_path = File.join(Pod::Config.instance.installation_root, react_native_path, "package.json")
+  package = JSON.parse(File.read(package_path))
```

Comment:
As done like this, it breaks RNTester, unfortunately. Should we find a better way to handle this? 🤔

Reply:
How does it break RNTester? I printed out the output and it looks correct to me?

```diff
 Generating Pods project
+"/~/react-native/packages/rn-tester/../../package.json"
 Setting REACT_NATIVE build settings
 Setting CLANG_CXX_LANGUAGE_STANDARD to c++17 on /~/react-native/packages/rn-tester/RNTesterPods.xcodeproj
 Pod install took 12 [s] to run
 Integrating client project
 Pod installation complete! There are 67 dependencies from the Podfile and 55 total pods installed.
```

Reply:
my bad the RNTester and CircleCI issue is due to hermes-engine versions... 🤦